Envoy.blade.php

```diff
@@ -210,6 +213,43 @@
     fi
 @endtask
 
+@task('deploy:cronjobs')
+    IS_CRONTAB_MANAGED_BY_ENVOY=$(crontab -l | grep -c 'MANAGED BY ENVOY') || true
+    IS_CRONTAB_EMPTY=$(crontab -l | wc -c) || true
+
+    @if (is_array($cronJobs))
+        if [ "$IS_CRONTAB_MANAGED_BY_ENVOY" -eq 0 ] && [ "$IS_CRONTAB_EMPTY" -ne 0 ]; then
+            echo 'SOME CRON JOBS ARE ALREADY INSTALLED AND ARE NOT MANAGED BY ENVOY!' >&2
+            echo 'To allow envoy to run, either copy the existing cron jobs to the configuration file or remove the cron jobs manually.' >&2
+            false
+        fi
+
+        @if (count($cronJobs) > 0)
+            FILE=$(mktemp)
+
+            echo '# WARNING! THIS FILE IS MANAGED BY ENVOY. DO NOT EDIT!' >> $FILE
+            echo 'SHELL="/bin/bash"' >> $FILE
+            echo 'MAILTO="{{ $cronMailTo }}"' >> $FILE
+            echo >> $FILE
+
+            @foreach ($cronJobs as $cronJob)
+                echo {{ escapeshellarg($cronJob) }} >> $FILE
+            @endforeach
+
+            crontab $FILE
+            rm $FILE
+        @else
+            crontab -r || true
+        @endif
+    @else
+        if [ "$IS_CRONTAB_MANAGED_BY_ENVOY" -ne 0 ]; then
+            echo 'YOU FORGOT TO REMOVE THE CRON JOBS THAT WERE INSTALLED WITH ENVOY!' >&2
```

Comment:
This message is not clear. The user does not necessarily have forgotten to remove the cron job : it may already be there due to another deployment.